rollup.config.js

```diff
@@ -19,7 +20,33 @@ export default {
 		}),
 		json(),
 		svelte({
+
 			exclude: `**/static-html/**/*.html`,
+
+			preprocess: {
+				style({ content }) {
+					return new Promise(fulfill => {
```

Comment:
Looks legit to me.  One possible change I see: since `process()` returns a thenable, you can resolve it directly with [`Promise.resolve`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve) instead of constructing your own Promise.  

The main advantage there is that all errors/responses are represented in the new Promise, whereas right now this promise is only getting resolved if execution makes it all the way down to line 40 without any errors happening.

```js
style({ content }) {
	return Promise.resolve(postcss([
		require(`precss`)({
			import: {
				path: [ `client/global-css` ],
				prefix: ``,
			},
		}),
		require(`autoprefixer`),
	])
		.process(content)
		.then(result => {
			return { code: result }
		}))
},
```